packages/server/src/subscriptions/websockets.ts

```diff
@@ -151,6 +151,8 @@ export async function handleR4SubscriptionConnection(socket: ws.WebSocket): Prom
     }
     subscribeWsToSubscription(socket, subscriptionId);
     ensureHeartbeatHandler();
+    // Send a handshake to notify client that this subscription is active for this connection
+    socket.send(JSON.stringify(createHandshakeBundle(subscriptionId)));
```

Comment:
Does this require that clients and server are on same version?

In other words, will this break clients on older versions?

Reply:
Technically it will still work but it won't be filtered out, so users would get it as a bundle in their callback and would have to filter it out

This could be seen as a breaking change in that regard, though this is actually compliant with the FHIR spec and we should probably do it for more consistent behavior with disconnecting / reconnecting logic 🤔 
https://build.fhir.org/ig/HL7/fhir-subscription-backport-ig/channels.html#:~:text=single%20websocket%20connection.-,Server%20sends%20one%20or%20more%20handshake%20messages%20via%20websockets%20(one%20per,last%20%E2%80%98n%E2%80%99%20messages%2C%20etc.).%20Clients%20are%20expected%20to%20handle%20either%20flow.,-Server%20may%20send

We might need to check with some of the clients who are already using it in prod?